<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_worker</class>
 <widget class="QMainWindow" name="add_worker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление сотрудника</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="full_name_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ФИО:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="full_name_pole">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="image_browse_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>112</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="age_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Возраст:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="age_pole">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="image_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фото:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stat_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести</string>
    </property>
   </widget>
   <widget class="QLabel" name="way_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
